README.md

```diff
@@ -32,6 +33,17 @@ You can find more information and talk to developers (let us know what you're wo
 In the past, we developed prototypes for semantic segmentation and tagging in this repo, which were discussed in our [segmentation ](https://www.azavea.com/blog/2017/05/30/deep-learning-on-aerial-imagery/), and [tagging](https://www.azavea.com/blog/2018/01/03/amazon-deep-learning/) blog posts. This implementation has been removed from the `develop` branch and is unsupported, but can still be found at [this tag](https://github.com/azavea/raster-vision/releases/tag/old-semseg-tagging).
 Similarly, an outdated prototype of object detection can be found at [this tag](https://github.com/azavea/raster-vision/releases/tag/old-object-detection) under the `rv` module.
 
+### Docker images
+
+Raster Vision is publishing docker images to [quay.io](https://quay.io/repository/azavea/raster-vision).
+The tag for the `raster-vision` image determines what type of image it is:
+- The `cpu-*` tags are for running the CPU containers.
+- The `gpu-*` tags are for running the GPU containers.
+
+We publish a new tag per commit to `develop`, which is tagged with the commit message.
```

Comment:
Seems like overkill to publish a tag for each commit -- once per PR seems like enough. Also, we should probably include the RV source code in the Docker image which we're not currently doing.

Reply:
It's per merge commit, i.e. merge of a PR. I'll change the language.

Including the source is not how RV currently operates - the task checks out the specific branch that you specify with the --remote flag. Are you suggesting to change the behavior? That might be better in a subsequent PR.

Reply:
I still think we should mount a specific commit of RV to the container, but that maybe the image should have RV baked into it as well so that RV can be run in the image without having to mount a volume containing the code. It depends on whether the image is supposed to be self-contained, or if it's just supposed to have the dependencies for RV. It'd be an easy thing to add later.